.env

```diff
@@ -1 +1,7 @@
+REDIS_URL=redis://localhost:6379
+DATABASE_SERVER="postgresql://postgres@localhost:5432"
+SEED_ADMIN_EMAIL=admin@example.com
+SEED_ADMIN_INITIAL_PASSWORD=password
+SIDEKIQ_PASSWORD=password
+SIDEKIQ_USERNAME=abtion
```

Comment:
The `dotenv-rails` gem is only loaded for envs `development` and `test`.

So we can safely store shared vars in here without risking them showing up in production.